sim/src/main/scala/spinal/sim/VCSBackend.scala

```diff
@@ -0,0 +1,377 @@
+package spinal.sim
+
+import org.apache.commons.io.FileUtils
+import spinal.sim.vpi.SharedMemIface
+
+import java.io.{File, PrintWriter}
+import java.nio.file.{Files, Paths}
+import scala.sys.process.{Process, ProcessLogger}
+
+import scala.collection.mutable
+case class VCSFlags(
+                     vloganFlags    : mutable.ArrayBuffer[String],
+                     vhdlFlags    : mutable.ArrayBuffer[String],
+                     elaborateFlags    : mutable.ArrayBuffer[String],
+                     runFlags    : mutable.ArrayBuffer[String]
+                   ) {
+  private def addFlags(container: mutable.ArrayBuffer[String], flags: String*): this.type = {
+    flags.foreach {f=>
+      container ++= f.split(" ")
+    }
+    this
+  }
+
+  def withDefaultVloganFlags(): this.type = {
+    vloganFlags ++= List(
+      "-nc",
+      "-full64",
+      "-sverilog",
+      "+v2k",
+      "-ntb",
+      "-debug_access+all",
+      "-timescale=1ns/1ps",
```

Comment:
Should we be setting a default precision (and scale)? Shouldn't the sim take the highest precision from the source files?

Reply:
oops, I forgot to remove these lines.

Reply:
Removed in https://github.com/SpinalHDL/SpinalHDL/pull/911/commits/179e86f91bfe6a9a768a7aea26e1b02682400e02#diff-7d9bf2ef5d69afec7b529e62bdc45c2de4fddaaf7ecd156895a5a4740cd6c5d5